src/js/components/molecules/wizard-step-download-list/wizard-step-download-list.unit.spec.ts

```diff
@@ -26,6 +26,15 @@ describe('components', () => {
                     expect(vm.$el).toBeDefined();
                     vm.$destroy();
                 });
+
+                it('should change the download url', () => {
```

Comment:
I would expect a spy on `location.assign` and for that spy to have been called with `data[key].items[1].file` (which could be destructured for readability maybe).

Reply:
I use the form action now to init the download, works without js because it is a submit button.